models/gc_picai_baseline/meta.json

```diff
@@ -0,0 +1,138 @@
+{
+  "id": "c5f886fb-9f54-4555-a954-da02b22d6d3f",
+  "name": "picai_baseline",
+  "title": "PI-CAI challenge baseline",
+  "summary": {
+    "description": "The PI-CAI challenge is to validate modern AI algorithms at clinically significant prostate cancer (csPCa) detection and diagnosis. This model algorithm provides the baseline for the challenge.",
```

Comment:
As discussed, we shall describe the model > challenge.

Reply:
You can use the description from the GC page:

This algorithm predicts a detection map for the likelihood of clinically significant prostate cancer (csPCa) using biparametric MRI (bpMRI). The algorithm ensembles 5-fold cross-validation models that were trained on the PI-CAI: Public Training and Development Dataset v2.0. The detection map is at the same spatial resolution and physical dimensions as the input axial T2-weighted image.

Reply:
Thanks! We have used your suggested description.